### DISCUSSION TITLE
Quick edit (list/related list) > return to initial view

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16036] [Feature improvement]
After clicking quick edit icon, and saving record, the record stays in detail view. 
It would be logical if after save, user is getting back to initial view,. In current design Quick Edit is not that quick :-)

![image](https://user-images.githubusercontent.com/84110093/167540734-22ee8c73-7d29-4ca1-9d7f-2bae573fb8d7.png)

best

**Comment by apcloic** [comment]
@s2b-git
Tested with module Tickets (and related contacts) and also with module Occurrences (and related participants) on [gitdeveloper ](https://gitdeveloper.yetiforce.com/)
User is getting back to initial view as expected.
Regards,

**Comment by s2b-git** [comment]
Certainly you misunderstood. See gif

![quick-edit](https://user-images.githubusercontent.com/84110093/167754052-d2c69580-b5d8-448e-bb53-b671d82b2cf3.gif)

**Reply by apcloic** [reply]
@s2b-git I don't think I misunderstood :)
Your question is about quick edit but you did click on 'Edit' button not on 'Quick Edit'

![image](https://user-images.githubusercontent.com/2930560/167897203-e4f11747-cce9-460a-91d5-f6b82278fb87.png)


![quick_edit_relatedlist](https://user-images.githubusercontent.com/2930560/167896554-e77e3e1a-b4a5-4b74-997c-7da857f725d4.gif)

**Comment by s2b-git** [comment]
You are right. What happen here the modules I tested are turned to advanced and basically I don see this icon... sorry for noise :-)
